### PR TITLE
Apply POM conventions and cleanup POMs

### PR DESCRIPTION
**Pull request by wborn**
Apply conventions with sortpom-maven-plugin:

    mvn com.github.ekryd.sortpom:sortpom-maven-plugin:sort -Dsort.keepBlankLines=true

Cleanup:

* Remove inherited groupId, version properties
* Remove unused bundle.symbolicName and bundle.namespace properties
* Convert tabs in POMs back to spaces (OnlyTabIdentationInXmlFilesCheck is suppressed for POMs in SAT 0.5.0 (https://github.com/openhab/static-code-analysis/pull/271, https://github.com/openhab/static-code-analysis/pull/289)

---

See also: https://github.com/eclipse/smarthome/pull/5338

### CODE REVIEW

File: addons/binding/org.openhab.binding.autelis/pom.xml [hunk 1]
```diff
@@ -1,18 +1,17 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.autelis/pom.xml]: Just a quick check is this how the skeleton makes them nowadays?

Comment (wborn) [reply addons/binding/org.openhab.binding.autelis/pom.xml]: In the near future when https://github.com/eclipse/smarthome/pull/5380 is merged. :slightly_smiling_face:

**Comment by pfink** [comment]
@martinvw @kaikreuzer @wborn: Please always ping me on POM refactoring PRs, we have to test those within the release pipeline before merging those. I took me hours for finding the bug caused by this PR :(

This PR changed the groupId of some artifacts. E.g. for artifact `org.openhab.io.transport.feed`:

groupId before PR: `org.openhab.io`
groupId after PR:  `org.openhab.addons.io`

This change was not applied all references to those artifacts and is causing build errors now. Snapshotbuild is not affected because just the old snapshots are taken. How should we move on here? Recover the groupId's before the PR or updating all references with the new groupId?

**Comment by martinvw** [comment]
@pfink apologies I don’t think this was intended but @wborn would know.

**Comment by kaikreuzer** [comment]
> How should we move on here? Recover the old groupId's or updating all references with the new groupId's?

I think the change clearly made sense (as it made the group ids the same for all bundles in the same folder) - so clearly, the references should be updated.

Could you create an according PR? (As you didn't give details, I probably won't find all references that you came across)

**Comment by pfink** [comment]
Are you sure? I think currently it's a little bit inconsistent:

Artifacts within the path addons/binding would have

- groupId: `org.openhab.binding`
- artifactId: `org.openhab.binding.bindingname`

Artifacts within the path addons/io would have

- groupId: `org.openhab.addons.io`
- artifactId: `org.openhab.io.BINDINGNAME` (without `addons`)

So the groupIds of different addon types would have a different scheme and the groupId of io addons would not correspond to it's artifactId. I'd suggest to either:

- change groupId's **and** artifactId's of all addons according to the new scheme: `org.openhab.addons.TYPE.NAME` or
- fully recover the old groupId scheme (`org.openhab.TYPE.NAME`) but keep the advantages of this PR (inherited groupId's)

**Comment by wborn** [comment]
Sorry for any inconveniences caused by this PR @pfink! It has gone unnoticed that these group naming inconsistencies existed. I'll also request reviews from you for future POM PRs. 

The voice addons also have a parent with addons in the groupId.

I think it's better to remove `addons` from the groupId because it is also not part of the bundle and package names.

**Comment by kaikreuzer** [comment]
See https://github.com/openhab/openhab2-addons/pull/3550.

It seems that the features already referenced those group ids, so I'd hope that this PR fixes the issue fully?

**Comment by pfink** [comment]
I think so!

**Pull request merged**